Solutions/Jamf Protect/Analytic Rules/JamfProtectUnifiedLogs.yaml

```diff
@@ -0,0 +1,61 @@
+id: 9eb2f758-003b-4303-83c6-97aed4c03e41
+name: Jamf Protect - Unified Logs
+description: |
+  'Creates an informational incident based on Jamf Protect Unified Log data in Microsoft Sentinel'
+severity: Informational
+status: Available
+requiredDataConnectors:
+  - connectorId: JamfProtect
+    dataTypes: 
+      - jamfprotect_CL
+suppressionDuration: PT5H
+suppressionEnabled: false
+tactics:
+relevantTechniques:
+query: |
+  jamfprotect_CL
+  | where input_eventType_s == "GPUnifiedLogEvent"
+  | extend severity = case(input_match_severity_d == 0, "Informational", input_match_severity_d == 1, "Low", input_match_severity_d == 2, "Medium", input_match_severity_d == 3, "High", "Informational")
+  | where isnotempty(severity)
```

Comment:
This condition will always be true as we are assigning a default value of "Informational".